_posts/2024-08-15-omero-5-6-13.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.13
+intro-blurb: The OME team is pleased to announce the release of OMERO.server 5.6.13
+---
+
+Today we are releasing OMERO.server 5.6.13 which includes:
+
+- This release reverts a change to the build process introduced in 5.6.12 that resulted in
+unused copies of old jars being included in the release. No other changes have been
+made to the software.
+
+**Note:** This upgrade will not invalidate the Bio-Formats Memoizer cache. Please
```

Comment:
I wonder if we need to rethink this message. This is only true if you upgraded from OMERO.server 5.6.12 (and regenerated the Bio-Formats memo files). For any upgrade from an earlier version of OMERO.server, the memo files will be invalidated by the Bio-Formats version bump.

Reply:
The OMERO.server 5.6.12 release https://www.openmicroscopy.org/2024/07/29/omero-5.6.12.html also says "This upgrade will not invalidate the Bio-Formats Memoizer cache"

Reply:
Yep, same problem, that's only true if you upgrade from OMERO.server 5.6.11

Reply:
Added clarification below